### DISCUSSION TITLE
Prism issues after deployment

### DISCUSSION THREAD

**Discussion by ChangJuneKim** [Milkdown/milkdown#887] [Q&A] [answered]
There is no problem locally, but an error occurs after deployment
![image](https://user-images.githubusercontent.com/77381154/215625973-ff7c340e-a9a5-4dcd-9562-abeedc5cfc37.png)
plz help...

milkdown version
"@milkdown/core": "^6.5.4",
"@milkdown/plugin-block": "^6.5.4",
"@milkdown/plugin-cursor": "^6.5.4",
"@milkdown/plugin-emoji": "^6.5.4",
"@milkdown/plugin-history": "^6.5.4",
"@milkdown/plugin-indent": "^6.5.4",
"@milkdown/plugin-listener": "^6.5.4",
"@milkdown/plugin-menu": "^6.5.4",
"@milkdown/plugin-prism": "^6.5.4",
"@milkdown/plugin-slash": "^6.5.4",
"@milkdown/plugin-tooltip": "^6.5.4",
"@milkdown/preset-commonmark": "^6.5.4",
"@milkdown/preset-gfm": "^6.5.4",
"@milkdown/prose": "^6.5.4",
"@milkdown/react": "^6.5.4",
"@milkdown/theme-nord": "^6.5.4",

**Comment by Saul-Mirone** [comment, marked as answer]
It's a issue caused by refactor, you need to configure the prism plugin like here:
https://github.com/Saul-Mirone/milkdown/blob/v6/website/component/MilkdownEditor/docRendererFactory.ts#L34

**Reply by ChangJuneKim** [reply]
thanks for your help!
